experimental/quarkus/image/project/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ## Stage 1 : build with maven builder image with native capabilities
-FROM quay.io/quarkus/centos-quarkus-maven:19.2.1 AS build
```

Comment:
I think the build is failing here because you need to specify a java version. See https://quay.io/repository/quarkus/centos-quarkus-maven?tag=19.2.1&tab=tags

Reply:
Thanks for spotting that. I think it's fixed now.